contrib/website/assets/styles/main.css

```
@@ -0,0 +1,1325 @@
+    transform-origin: center bottom;
+    animation: eyeBounce 1.2s cubic-bezier(0.36, 0.07, 0.19, 0.97) 1 both;
+}
+
+#eye1 {
+    animation-delay: 0.1s;
+}
+#eye2 {
+    animation-delay: 0.22s;
+}
+
+@keyframes eyeBounce {
```

Stylelint (17.9.0)

[error] Expected keyframe name "eyeBounce" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)